core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/KubernetesClient.scala

```diff
@@ -93,10 +94,15 @@ class KubernetesClient(
     with ProcessRunner {
   implicit protected val ec = executionContext
   implicit protected val am = ActorMaterializer()
+  protected val configBuilder = new ConfigBuilder()
```

Comment:
As `configBuilder` is not to be used further we can avoid making it a instance field. So we can refactor this a bit via

```scala
    val builder = new ConfigBuilder()
      .withConnectionTimeout(config.timeouts.logs.toMillis.toInt)
      .withRequestTimeout(config.timeouts.logs.toMillis.toInt)
    config.actionNamespace.foreach(builder.withNamespace)
    new DefaultKubernetesClient(builder.build())
  }
```